linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/SJSGen.scala

```diff
@@ -112,11 +112,18 @@ private[emitter] final class SJSGen(
         args.toList)
   }
 
-  def usesUnderlyingTypedArray(elemTypeRef: NonArrayTypeRef): Boolean =
-    getArrayUnderlyingTypedArrayClassRef(elemTypeRef)(Position.NoPosition).nonEmpty
+  def usesUnderlyingTypedArray(elemTypeRef: NonArrayTypeRef): Boolean = {
+    /* We are only interested in whether `getArrayUnderlyingTypedArrayClassRef`
+     * returns a `Some` or not. We do not keep the result, so the `Position`
+     * and the `GlobalRefTracking` are irrelevant.
+     */
```

Comment:
:facepalm: I could have figured that out. But the comment for sure is still a good idea :P